test/models/index.ts

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'chai';
+import { loadServiceConfigsFromFile, getServiceConfigs, validateServiceConfig } from '../../app/models/services';
+import _ from 'lodash';
+
+describe('Services.yml validation', function () {
+
+  describe('harmony services.yml configuration is valid', function () {
```

Comment:
This is such a great test that I wish we had added a long time ago. Can you change it to make sure it validates that all of the configuration for both Prod and UAT are correct?

It will require refactoring `loadServiceConfigs()` to take in the CMR environment URL as an argument and will need to return the service configurations from it, but then we'll be guaranteed that services.yml is correct in all environments.